GoogleSignIn/Tests/Unit/GIDGoogleUserTest.m

```diff
@@ -34,10 +35,23 @@
 #import <AppAuth/OIDAuthState.h>
 #endif
 
+static NSString *const kNewAccessToken = @"new_access_token";
+static NSTimeInterval const kAccessTokenExpireTime = 442886100;
+static NSTimeInterval const kIDTokenExpireTime = 442886118;
+static NSTimeInterval const kNewAccessTokenExpireTime = 442886200;
```

Comment:
I don't think these constants are used outside of `testUpdateAuthState` below. Do you have plans to use them in future tests? If not, then maybe they don't need to be constants right now? 

This comment is driven by my concern that the constants don't really have meaningful names (or meaning in general) outside of `testUpdateAuthState`. 

You could create `NSTimeInterval`s within the method and give them good names in the new test method. Then, you could create new intervals with better names than "new" (e.g., something like `updatedAccessTokenExpireTime`) in the test method. 

[NSDate has a bunch of methods](https://developer.apple.com/documentation/foundation/date/1779636-timeintervalsince) that you can use to create time intervals.

PS I get the impression that you're following a pre-existing pattern in [GIDAuthenticationTest](https://github.com/google/GoogleSignIn-iOS/blob/main/GoogleSignIn/Tests/Unit/GIDAuthenticationTest.m), but IMHO it's not very readable there either.

Reply:
These value will be used in other test cases. But I think creating them inside the tests make the code easier to understand. Thanks for pointing it out.